gettsim/data/unterhalt.yaml

```
@@ -1,51 +1,107 @@
-uhv5:
+uhv6_share:
   name:
-    de: Unterhaltsvorschuss für Kinder unter 5 Jahren. Der Prozentsatz in §2 UVG wird angewendet auf (2 x Kinderfrebeitrag (32 (6) EStG) / 12) - Kindergeld für das 1. Kind
+    de: Unterhaltsvorschuss für Kinder bis 6 Jahre. Anteil am Kinderfreibetrag.
     en:
   description:
-    de: §2 UVG, §1612a BGB
+    de: §1612a (1) Nr.1 BGB
     en:
   values:
-    2015-01-01:
-      value: 322.5
-      note: Ausnahmsweise §11a UHV, Durchschnitt aus beiden Werten
+    2008-01-01:
+      value: .87
+      note: Artikel 1 G. v. 21.12.2007, BGBl. I S. 2631
+uhv12_share:
+  name:
+    de: Unterhaltsvorschuss für Kinder von 7-12 Jahren. Anteil am Kinderfreibetrag.
+    en:
+  description:
+    de:  §1612a (1) Nr.2 BGB
+    en:
+  values:
+    2008-01-01:
+      value: 1
+      note: Artikel 1 G. v. 21.12.2007, BGBl. I S. 2631
+uhv17_share:
+  name:
+    de: Unterhaltsvorschuss für Kinder ab 13 Jahren. Anteil am Kinderfreibetrag.
+    en:
+  description:
+    de: §1612a (1) Nr.3 BGB
+    en:
+  values:
+    2008-01-01:
+      value: 1.17
+      note: Artikel 1 G. v. 21.12.2007, BGBl. I S. 2631
+uhv6_amount:
+  name:
+    de: Mindestunterhalt für Kinder bis 6 Jahre
+    en:
+  description:
+    de: §1 Mindesunterhaltsverordnung (MUV)
+    en:
+  values:
+    2016-01-01:
+      value: 335
+      note: G. v. 03.12.2015, BGBl. I S. 2188
     2017-01-01:
-      value: 154
-      note:
+      value: 342
+      note: G. v. 03.12.2015, BGBl. I S. 2188
+    2018-01-01:
+      value: 348
+      note: Artikel 1 G. v. 28.09.2017, BGBl. I S. 3525
     2019-01-01:
-      value: 160
-      note:
-uhv11:
+      value: 354
+      note: Artikel 1 G. v. 28.09.2017, BGBl. I S. 3525
```

```suggestion
      note: Artikel 1 V. v. 28.09.2017 BGBl. I S. 3525.
```
it is a Verordnung, not Gesetz. Also remove the comma to make it slightly easier to search for on buzer.de and on offenegesetze.de.

This probably applies to more entries.

---

Is that even the correct decree for the 2019-01-01 policy date? The decree it is from 2017.

---

OK. not sure whether the full stop makes things easier though.

---

The full stop makes no difference for the search bar over on offenegesetze.de.

In some cases there is additional info (maybe an explation) on the note key (e.g. [here](https://github.com/iza-institute-of-labor-economics/gettsim/blob/79ca7cedd79566c453f41d4bd6ddddfb81731219/gettsim/data/arbeitsl_geld_2.yaml#L273) where I added a note). I think that is why I added a full stop everywhere. I have no strong opinion on omitting it when there is nothing more in the note.

---

Should we leave it open for now and at some point define guidelines for the param files?

---

Looking at [Artikel 1 V. v. 28.09.2017 BGBl. I S. 3525](https://offenegesetze.de/veroeffentlichung/bgbl1/2017/66#page=13) again: You are right. It defines both 2018-01-01 and 2019-01-01.

I think we should not leave a "G. v." there as it is a "V. v." ;) Simply is wrong. Nothing to be defined in note style guidelines. 

I have not much to say about note style guidelines. This really is a detail. We can have a separate discussion on that, sure (about the comma, the full stop and things like "Artikel.."). [Here](https://github.com/iza-institute-of-labor-economics/gettsim/pull/69) I basically just follow the principle to make it is as easy as possible to look up the parameter in the law over on offenegesetze.de and buzer.de plus just have a consistent citing style (it is the citing style  buzer.de uses btw. So it is even very easy to copy and paste the citation).

---

Well, that sounds like a guideline, but it would need to be made consistent in all files. Actually, in order to make full-text search work, one would need to replace the month to words. ('September' instead of '.09.'). Using this bit: `2017 BGBl. I S. 3525` should however lead to the correct bit.

---

OK, let's discuss that at https://github.com/iza-institute-of-labor-economics/gettsim/issues/127